iroh/src/client.rs

```diff
@@ -104,37 +178,99 @@ where
         };
         Ok(Some(doc))
     }
+}
+
+/// Iroh blobs client.
+#[derive(Debug, Clone)]
+pub struct BlobsClient<C> {
+    rpc: RpcClient<ProviderService, C>,
+}
+
+impl<C> BlobsClient<C>
+where
+    C: ServiceConnection<ProviderService>,
+{
+    /// Read a single blob as a streaming [`BlobReader`].
```

Comment:
I would describe this just as as `Stream the contents of a blob` or sth like that. The `BlobReader` is not really important, as it is just the type for the stream.